### PR TITLE
update: add directories

### PR DESCRIPTION
**Pull request by kalanchan**

### CODE REVIEW

**Review by daxmc99 — APPROVED** [review]

File: content/departments/engineering/engineering/admin-exp/devops/managed/index.md [hunk 1]
```diff
@@ -164,6 +164,19 @@ Terraform is used to maintain all managed instances. You can find this configura
 
 All customer credentials, secrets, site configuration, app and user configuration—is stored in Postgres only (i.e. on the encrypted GCP disk). This allows customers to enter their access tokens, secrets, etc. directly into the app through the web UI without transferring them to us elsewhere.
 
+### Operations
+
+Managed Instances v1.1 documentation can be found [here](./v1.1/index.md)
+
+- [Creating a Managed Instance](./creation_process.md)
```
Comment (michaellzc) [inline content/departments/engineering/engineering/admin-exp/devops/managed/index.md]: we already have a ToC here https://github.com/sourcegraph/handbook/blob/main/content/departments/engineering/engineering/admin-exp/devops/managed/operations.md

can we just point user to the above page?

I am hoping we can keep this page cleaner and the target audience is people outside of our team

Comment (michaellzc) [reply content/departments/engineering/engineering/admin-exp/devops/managed/index.md]: or maybe we can crates a new operation folder and nest all processes under it

Comment (kalanchan) [reply content/departments/engineering/engineering/admin-exp/devops/managed/index.md]: oooh, great call @michaellzc. I didn't even see the operations hyperlink, what do you think about keeping the `operations` header and providing a link to `operations.md` instead

Comment (michaellzc) [reply content/departments/engineering/engineering/admin-exp/devops/managed/index.md]: I am fine with it.

Maybe we can also add them here so it's more visible. https://github.com/sourcegraph/handbook/blob/main/content/departments/engineering/engineering/admin-exp/devops/managed/index.md?plain=1#L9-L19

File: content/departments/engineering/engineering/admin-exp/devops/managed/v1.1/index.md [hunk 2]
```diff
@@ -28,3 +28,10 @@ _To update this file: goto
 
 [cloud sql]: https://cloud.google.com/sql/postgresql
 [google cloud storage]: https://cloud.google.com/storage
+
+### Operations
+
+- [Upgrade a Managed Instance](./mi1-1_upgrade_process.md)
+- [Restore a Managed Instance](./mi1-1_restore_process.md)
+- [Failover Process](./mi1-1_failover_process.md)
+- [Enable Executors](./mi1-1_enable_executors_process.md)
```
Comment (michaellzc) [inline content/departments/engineering/engineering/admin-exp/devops/managed/v1.1/index.md:36]: Are you intended to nest `Operations` under the `Technical details` header?

Also, L29-L30 are actually link definitions and usually they go to the bottom, i.e. moving `operations` above L29.

**Review by michaellzc — APPROVED** [review]

**Pull request merged**